config/main.py

```diff
@@ -3916,6 +3916,172 @@ def del_user(db, user):
             click.echo("Restart service snmp failed with error {}".format(e))
             raise click.Abort()
 
+#
+# 'bmp' group ('config bmp ...')
```

Comment:
>bmp [](http://example.com/codeflow?start=3&length=3)

Offline discussed, we need a feature level on/off. Some user may choose to disable the feature completely and do not waste any RAM or CPU.

Reply:
Added in https://github.com/sonic-net/sonic-buildimage/pull/18946/files#diff-94736fc06ef95ff9c8dc4dd4088f87f1c62ad322286b40cfd5b1b6c75bcb3764

Reply:
No, your link is build-time flag. I am talking about runtime enable/disable.

Reply:
also need testcase for bmp feature enable/disable

Reply:
I see runtime flag. How about testcase for bmp feature enable/disable?

Reply:
Tracked via https://msazure.visualstudio.com/One/_workitems/edit/27588923